MekHQ/src/mekhq/MekHQ.java

```diff
@@ -366,8 +366,12 @@ public static void main(String[] args) {
         System.setProperty("com.apple.mrj.application.apple.menu.about.name","MekHQ");
 
         SwingUtilities.invokeLater(() -> {
-            //redirect output to log file
-            redirectOutput();
+            // TODO : logFileName should probably not be inline
+            String logFileName = "logs" + File.separator + "mekhqlog.txt";
+            // Reset the log file - TODO : Maybe move this to being a call to the logger?
+            MegaMek.resetLogFile(logFileName);
```

Comment:
Should this be added to the logger?

Reply:
I don't know but I'm confused as to why the log file bits have to be invoked on the UI thread.

Reply:
Good point, I'll test and see what happens outside the UI thread.

Reply:
I've got no idea why it was in the UI thread, but I've moved it to its over thread and added a reset log file method to the default logger.